auth-ii_client/src/users/Users.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import { Button } from 'reactstrap';
+import { Redirect } from 'react-router';
+
+
+class Users extends Component {
+  state = {
+    users: [],
+    LoggedIn: true
+  }
+
+  render() {
+    const redirectToHome = this.state.LoggedIn;
+    if (redirectToHome === false) {
+      return (<Redirect exact to='/' />)
```

Comment:
Nice job! There are two ways to go about this, one is using a `<Redirect />` and another one `history.push`. But the `<Redirect />` is usually the favored.